week-1-practise/level-3/1-piping.js

```diff
@@ -26,6 +25,14 @@ function multiply(a, b) {
 function format(num) {
   return "£" + num.toFixed(2);
 }
+var startingValue = 5;
+
+
+
+
+let result = format(add(multiply(2,5),10));
```

Comment:
Hmm, almost there, but let's use startingValue variable!

Reply:
You mean instead of using 5 , assign 5 to a variable such as startingValue and then use that variable within the formula as fellow

let result = format(add(multiply(2,startingValue),10));

is that what you mean?